implementation/src/main/module/module-info.java

```diff
@@ -1,6 +1,17 @@
 open module io.smallrye.mutiny {
 
-    requires org.reactivestreams;
+    requires transitive org.reactivestreams;
+    requires transitive smallrye.common.annotation;
+
+    exports io.smallrye.mutiny;
+    exports io.smallrye.mutiny.groups;
+    exports io.smallrye.mutiny.helpers.spies;
+    exports io.smallrye.mutiny.helpers.test;
+    exports io.smallrye.mutiny.infrastructure;
+    exports io.smallrye.mutiny.operators;
```

Comment:
Should it be exposed? 
I'm not sure we capture breaking changes of the operator package.

Reply:
That's a good question, I initially exposed it because of `AbstractMulti` / `AbstractUni` (`exports` clauses are not recursive so the actual operators won't leak into the public API).

Shall these be part of the public API? I'm fine reducing the scope.

Reply:
Yeah let's exclude and re-align with `revapi.json`